File: ShoppingELF/ShoppingELF/Models/SubCategoryModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ShoppingELF.Models
{
    public class SubCategoryModel
    {
        public int SubCategoryID { get; set; }
        public string SubCategoryName { get; set; }
        public int CategoryID { get; set; }

        public virtual CategoryTable CategoryTable { get; set; }
        public virtual ICollection<ProductTable> ProductTable { get; set; }
    }
}

File: Frontend/src/app/shared/SubCategory.model.ts
export class SubCategory
{
    SubCategoryID:number;
    SubCategoryName:string;
    CategoryID:number;


constructor( SubCategoryID:number,SubCategoryName:string,CategoryID:number)
{
    this.CategoryID=CategoryID;
    this.SubCategoryName=SubCategoryName;
    this.SubCategoryID=SubCategoryID;
}
}